src/assets/cmm_assessment.js

```diff
@@ -58,8 +58,17 @@ function saveFormValues() {
   });
 }
 
-function clearFormValues(model) {
-  localStorage.removeItem(model);
+function clearFormValues(item) {
+  localStorage.removeItem(item);
+}
+
+function clearCategoryValues(storageItem, category) {
+  // Get the current data object in storage
+  var data = JSON.parse(localStorage.getItem(storageItem));
+  // Remove the current category value from the object
+  delete data[category];
+  // Set the updated data object as the new cmm link
```

Comment:
comments seem over verbose
its quite clear what the code is doing?
```suggestion
  const data = JSON.parse(localStorage.getItem(storageItem));
  delete data[category];
```

Reply:
Ah I commented first as I was figuring out what the data object looked like, and then left it in there! Have removed now :)